@navikt/core/react/src/modal/dialog-polyfill.ts

```diff
@@ -675,7 +675,11 @@ dialogPolyfill.DialogManager.prototype.unblockDocument = function () {
 dialogPolyfill.DialogManager.prototype.updateStacking = function () {
   var zIndex = this.zIndexHigh_;
 
-  for (var i = 0, dpi; (dpi = this.pendingDialogStack[i]); ++i) {
+  for (
+    var i = 0, dpi: { updateZIndex: (arg0: number, arg1: number) => void };
+    (dpi = this.pendingDialogStack[i]);
+    ++i
+  ) {
```

Comment:
Selv liker jeg "plain for loops" (helst liker jeg vel ikke for loops heller, foretrekker for of/in 😅 ) og ikke gjøre for mye i selve for expression-loop-condition blocken (feks definere og sette flere variabler), all type "videre logikk" liker jeg da å ha i body til loopen i steden, og heller ta opp flere linjer 🤔 

Sier ikke at vi trenger refaktorere noe her (eller senere), bare merker jeg må tenke ekstra her 😅 (spesielt med ekstra type info!)

Reply:
Helt enig. Hele dialog-polyfillen er bare copy-paste fra en annen pakke, så tør ikke gjøre for mye endringer der. Hadde det vært opp til meg hadde jeg bare tatt en "biome-ignore" og latt det være 😅

Reply:
Hadde selv også lyst til å sette `biome-ignore` først, så hvis dere tenker det samme gjør vi det 🚀